server/handle_capabilities_test.go

```diff
@@ -246,61 +246,6 @@ func TestHandleCapabilities(t *testing.T) {
 				},
 			},
 		},
-		"config set hostname unset port": {
-			// With hostname set and port unset in config, urls should have host from config and
-			//  port from uri: "cdn.tegola.io:8080"
-			hostname: "cdn.tegola.io",
-			uri:      "http://localhost:8080/capabilities?debug=true",
-			expected: server.Capabilities{
-				Version: serverVersion,
-				Maps: []server.CapabilitiesMap{
-					{
-						Name:         "test-map",
-						Attribution:  "test attribution",
-						Center:       [3]float64{1.0, 2.0, 3.0},
-						Bounds:       tegola.WGS84Bounds,
-						Capabilities: "http://cdn.tegola.io:8080/capabilities/test-map.json?debug=true",
```

Comment:
If I understand this correctly, shoud this test case stay, but remove the `:8080` from urls?